### PR TITLE
using hyperkube for k8s-api-server

### PR DESCRIPTION
**Pull request by akanso**
addressing issue #135

### CODE REVIEW

**Comment by akanso** [comment]
Also added the defaults of **not** deploying mesos and swarm.

File: examples/envs/dev-vbox/radiant01.yml [hunk 1]
```diff
@@ -12,6 +12,11 @@
 # about these varaibles and ??? for the general story about HA
 # deployment.
 
+
+swarm_deploy: False
+mesos_deploy: False
+kubernetes_deploy: True
```
Comment (MikeSpreitzer) [inline examples/envs/dev-vbox/radiant01.yml]: I prefer to not make a copy of a default setting

File: examples/envs/dev-vbox/radiant01.yml [hunk 2]
```diff
@@ -12,6 +12,11 @@
 # about these varaibles and ??? for the general story about HA
 # deployment.
 
+
+swarm_deploy: False
```
Comment (MikeSpreitzer) [inline examples/envs/dev-vbox/radiant01.yml]: We are going to change the default for this, right?  No need to duplicate the setting here.

File: examples/envs/dev-vbox/radiant01.yml [hunk 3]
```diff
@@ -12,6 +12,11 @@
...
+mesos_deploy: False
```
Comment (MikeSpreitzer) [inline examples/envs/dev-vbox/radiant01.yml]: same as previous

**Comment by akanso** [comment]
@MikeSpreitzer then we can add the defaults in the group_vars/all 
I feel showing them in the radiant01.yml would give the users a clearer idea on how to configure things. The tiny example is already a specific example where we are defining a specific VIP, so we can specify other deployment options for clarity.

**Comment by MikeSpreitzer** [comment]
I think teaching people to put settings in their shard-specific variables files that are duplicates of default settings is a bad idea.  I have seen it happen in our earlier work with Kraken and Radiant.  The problem is that as the developers of the playbooks evolve what the desired defaults are, and evolve the set of variables that are used, there is a huge pile of cruft to maintain (and it is not always clear what the intent is).  How about putting in a comment instead of a duplicate setting?

**Comment by MikeSpreitzer** [comment]
I think we agreed earlier today on what to do.  We agreed to not duplicate material from `ansible/group_vars/all` into the shard files.  Changing the default for `swarm_deploy` was done in PR #141.  Factoring the Mesos support is something I will do soon.

**Pull request closed without merging**